src/components/Utils/search.ts

```diff
@@ -65,18 +65,50 @@ export const searchIndexes = (
     return results;
 };
 
+/**
+ * Search for a given pattern in the text
+ *
+ * @param rawKeywords - The Regex pattern to search.
+ * @param {Uint8Array} rawLog - The log data to search within.
+ * @returns {number[]} An array of indices where the keyword is found in the log.
+ */
+export const searchIndexesRegex = (
+    rawKeywords: string | undefined,
+    rawLog: Uint8Array
+) => {
+    if (!rawKeywords) return [];
+
+    const decodedLog = new TextDecoder("utf-8").decode(rawLog);
+    const indexes: number[] = [];
+
+    try {
+        const regex = new RegExp(rawKeywords, "g");
```

Comment:
Don't forget if caseInsensitive="true" you want to make this "gi"

Reply:
You right, fixed